src/commands/math/advancedSymbols.js

```diff
@@ -322,5 +323,11 @@ LatexCmds.cap = LatexCmds.intersect = LatexCmds.intersection =
 // FIXME: the correct LaTeX would be ^\circ but we can't parse that
 LatexCmds.deg = LatexCmds.degree = bind(VanillaSymbol,'\\degree ','&deg;');
 
+LatexCmds.celsius = bind(VanillaSymbol,'\\celsius ','&#8451;');
+LatexCmds.micro = bind(VanillaSymbol,'\\micro ','&#181;');
+LatexCmds.perthousand = bind(VanillaSymbol,'\\perthousand ','&#8240;');
+LatexCmds.ohm = bind(VanillaSymbol,'\\ohm ','&#8486;');
```

Comment:
Well, it depends which packages you want to support. They're in gensymb (ftp://ftp.dante.de/tex-archive/macros/latex/contrib/was/gensymb.pdf) and MathQuill already supports the \degree macro which is also in this package...